apprentice-react-o-shop/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Let's clean this file up and just have a one line description and steps to start your server